plugins.json

```diff
@@ -15723,6 +15723,99 @@
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-azure-devops-v3.2.0.tar.gz",
```

Comment:
out of curiosity (comparing this to autolink), why do we have much more data for azure than autolink (signatures, platform details, icon data, etc..)